build/crd-samples/router/create-rule-eventbus-rest.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rules.kubeedge.io/v1
+kind: Rule
+metadata:
+  name: my-rule-eventbus-rest
+  labels:
+    description: test
+spec:
+  source: "my-eventbus"
+  sourceResource: {"topic":"test","node_name":"xxx"}
+  target: "my-rest"
+  targetResource: {"resource":"http://a.com"}
```

Comment:
Do we need dns record for a.com?

Reply:
the a.com should be public address which cloudcore can access directly, if can't, we need add dns record.